inspirehep/modules/workflows/workflows/article.py

```diff
@@ -362,6 +367,7 @@ class Article(object):
         # use that instead of stopping
         CHECK_IF_MERGE_AND_STOP_IF_SO +
         CHECK_IF_SUBMISSION_AND_ASK_FOR_APPROVAL +
+        CHECK_IF_NOT_SUBMISSION_AND_ADD_HOLDINGPEN_ENTRY_ID +
```

Comment:
I would say, instead of creating a new step in the workflow, you could put this enhancement in POSTENHANCE_RECORD. In this way, we comply with the granularity of the current steps :)

Reply:
@iulianav what about this? I think now is the right moment to revive it :)